internal/pkg/docker/docker.go

```diff
@@ -56,19 +59,20 @@ func (r Runner) Login(uri, username, password string) error {
 	return nil
 }
 
-// Push will run `docker push` command against the Service repository URI with the input uri and image tag.
-func (r Runner) Push(uri, imageTag string) error {
-	path := imageName(uri, imageTag)
+// Push will run `docker push` command against the repository URI with the input uri and image tags.
+func (r Runner) Push(uri, imageTag string, additionalTags ...string) error {
+	for _, imageTag := range append(additionalTags, imageTag) {
+		path := imageName(uri, imageTag)
 
-	err := r.Run("docker", []string{"push", path})
-
-	if err != nil {
-		// TODO: improve the error handling here.
-		// if you try to push an *existing* image that has Digest A and tag T then no error (also no image push).
-		// if you try to push an *existing* image that has Digest B and tag T (that belongs to another image Digest A) then docker spits out an unclear error.
-		log.Warningf("the image with tag %s may already exist.\n", imageTag)
+		err := r.Run("docker", []string{"push", path})
+		if err != nil {
+			// TODO: improve the error handling here.
+			// if you try to push an *existing* image that has Digest A and tag T then no error (also no image push).
+			// if you try to push an *existing* image that has Digest B and tag T (that belongs to another image Digest A) then docker spits out an unclear error.
+			log.Warningf("the image with tag %s may already exist.\n", imageTag)
```

Comment:
Is this going to warn customers if an image with "latest" tag already exists in their repo?

Reply:
I was thinking that all the packages except `cli`, or `term/` can be consumed as a library by others. So there shouldn't be logging and strictly just passing data around. I'd in favor of removing this log statement.

Reply:
No I don't think so... in fact it seems to me that `docker push` won't throw an error if it's trying to push an `<image>:<tag>` while the `<tag>` already exists in the `repo /<image>` 👀

Reply:
@efekarakus agreed. I'm actually confused about why this log warning is placed within the error block since it seems that `docker push` won't complain if the image tag already existed. Is there any reason why we put this here initially?

Reply:
No idea :D let's remove it